### PR TITLE
Added tweets about browserlist

### PR DESCRIPTION
**Pull request by ariarzer**

### CODE REVIEW

File: client/view/Tweet/Tweet.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.Tweet {
+  position: relative;
+  z-index: 1;
+  height: fit-content;
+  padding: 16px;
+  border: 1px solid var(--separator);
+  border-radius: 12px;
+}
+
+.Tweet * {
```
Comment (ai) [inline client/view/Tweet/Tweet.css:10]:
```suggestion
.Tweet {
```

**Review by ai — APPROVED** [review]

**Comment by ai** [comment]
Resolve conflict and merge

**Comment by ariarzer** [comment]
Wait a second, i'd optimise images

**Comment by ariarzer** [comment]
Should i increase the size?...

**Comment by ai** [comment]
Yes. Increasing the size is not a problem. It is just a double-check that everyone is expecting the size increase (we added images do we expect it).

Note that our limit used 5 KB step to not increase on every small change.

File: client/.size-limit.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   {
     "name": "Files to download",
     "path": ["dist/**/*", "!dist/*.ico", "!dist/*.webmanifest", "!dist/*.png"],
-    "limit": "30 KB"
+    "limit": "31 KB"
```
Comment (ai) [inline client/.size-limit.json:5]:
```suggestion
    "limit": "35 KB"
```

5 KB step

**Pull request merged**